hooks/index.ts

```diff
@@ -2,3 +2,7 @@ export { default as useOgv } from "./useOgv";
 export { default as useViewWidth } from "./useViewWidth";
 export { default as useContracts } from "./useContracts";
 export { default as useInterval } from "./useInterval";
+export { default as useChartGradient } from "./useChartGradient";
+export { default as useGetPreviousRoute } from "./useGetPreviousRoute";
+export { default as usePreviousRoute } from "./usePreviousRoute";
```

Comment:
I wonder if we should start picking up a lib with a full set of these type of hooks like `react-use`?

Reply:
I didn't know about this... but def worth exploring when we're resolving tech debt. I see many of these custom hooks could be replaced with what the library provides.

Reply:
I didn't know about this... but def worth exploring when we're resolving tech debt. I see many of these custom hooks could be replaced with what the library provides.